app/models/patient.rb

```diff
@@ -90,6 +91,11 @@ class Patient < ApplicationRecord
             absence: { message: "cannot be 'true' when 'Last Date of Exposure' is specified" },
             if: -> { last_date_of_exposure.present? }
 
+  validates :follow_up_note,
+            on: %i[api import],
+            absence: { message: "must be blank when 'Follow-Up Reason' is blank" },
+            if: -> { follow_up_reason.nil? }
```

Comment:
This is pretty minor, but I kind of feel like the semantics of this would be better if this were instead a validation on `follow_up_field`, saying that it's required when a `follow_up_note` is present. My thinking is that if they put a follow-up note, it's more likely that they forgot to put a follow-up reason than mistakenly included the note.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/983/commits/fe4178fb3a2609750e4a208a3e9b506ca916b74c